public/documentation/tutorials/webassembly.md

```diff
@@ -0,0 +1,44 @@
+# Web Assembly
+
+WebAssembly or wasm is a new portable, size- and load-time-efficient format suitable for compilation to the web. It has following advantages:
+
+*   Efficient and fast.
+*   Safe.
+*   Open and debuggable.
+*   Part of the open web platform.
+
+WebAssembly is currently being designed as an open standard by a [W3C Community Group](https://www.w3.org/community/webassembly/) that includes representatives from all major browsers. Now it is available in Crosswalk 22 or later on Android, Windows and Linux. It's also [supported in multiple browsers](https://hacks.mozilla.org/2016/03/a-webassembly-milestone/)
+
+In the following tutorial, you'll learn how to enable WebAssembly in Crosswalk and run the demo [Angrybots](https://webassembly.github.io/demo/) on Windows platform.
+
```

Comment:
Here I just have Windows platform as an example. As the app-tools is cross-platform, the parameter is a little different from that of Windows. If you think it's necessary to list Windows/Android separately, I will add the Android part.